re.sonny.Workbench.Devel.json

```diff
@@ -71,6 +71,21 @@
         }
       ]
     },
+    {
+        "name" : "libshumate",
+        "buildsystem": "meson",
+        "config-opts": [
+            "-Dgtk_doc=false",
+            "-Dlibsoup3=true"
+        ],
+        "sources" : [
+            {
+                "type" : "git",
+                "url" : "https://gitlab.gnome.org/GNOME/libshumate.git",
+                "branch" : "main"
+            }
```

Comment:
We should use the latest stable version, please use the archive from download.gnome.org like we do for other GNOME modules.

Reply:
Also, it should be added to `re.sonny.Workbench.json` as well